pkg/cmd/runsGet.go

```diff
@@ -93,6 +94,8 @@ func (cmd *RunsGetCommand) createCobraCommand(
 	formatters := runs.GetFormatterNamesString(runs.CreateFormatters())
 	runsGetCobraCmd.PersistentFlags().StringVar(&cmd.values.runName, "name", "", "the name of the test run we want information about."+
 		" Cannot be used in conjunction with --requestor, --result or --active flags")
+	runsGetCobraCmd.PersistentFlags().StringVar(&cmd.values.group, "group", "", "the name of the group to return tests under that group."+
+		" Cannot be used in conjunction with --name")
```

Comment:
Just spotted this - if you run `galasactl runs get --group <group-here>`, the error that gets displayed is:

```
Error: GAL1079E: The --age or the --name parameter must be used to identify which test run(s) you want see. Use the --help flag for more information, or refer to the documentation at https://galasa.dev/docs/reference/cli-commands
```

but this mentions that `--group` cannot be used with `--name`, so the errors are conflicting

Reply:
Also, do we need to be insisting on `--age` being provided if we just want all runs for a given group?